ui.af.apps/src/main/content/jcr_root/apps/core/fd/components/form/scribble/v1/scribble/README.md

```diff
@@ -0,0 +1,102 @@
+
+### Use Object
+The Form Text component uses the `com.adobe.cq.forms.core.components.models.form.Scribble` Sling Model for its Use-object.
+
+### Edit Dialog Properties
+The following properties are written to JCR for this Form Text component and are expected to be available as `Resource` properties:
+
+1. `./jcr:title` - defines the label to use for this field
+2. `./hideTitle` - if set to `true`, the label of this field will be hidden
+3. `./name` - defines the name of the field, which will be submitted with the form data
+4. `./description` - defines a help message that can be rendered in the field as a hint for the user
+5. `./required` - if set to `true`, this field will be marked as required, not allowing the form to be submitted until the field has a value
+6. `./requiredMessage` - defines the message displayed as tooltip when submitting the form if the value is left empty
+
+## Client Libraries
+The component provides a `core.forms.components.scribble.v1.runtime` client library category that contains the Javascript runtime for the component. 
+It should be added to a relevant site client library using the `embed` property.
+
+It also provides a `core.forms.components.scribble.v1.editor` editor client library category that includes
+JavaScript handling for dialog interaction. It is already included by its edit dialog.
+
+## BEM Description
+```
+BLOCK cmp-adaptiveform-scribble
+    ELEMENT cmp-adaptiveform-scribble__label-container
+    ELEMENT cmp-adaptiveform-scribble__canvas-signed-container
+        ELEMENT cmp-adaptiveform-scribble__canvas-signed-image
+    ELEMENT cmp-adaptiveform-scribble__container
+        ELEMENT cmp-adaptiveform-scribble__header
+        ELEMENT cmp-adaptiveform-scribble__content
+            ELEMENT cmp-adaptiveform-scribble__canvases
+                ELEMENT cmp-adaptiveform-scribble__signcanvases
+                    ELEMENT cmp-adaptiveform-scribble__canvas
+                    ELEMENT cmp-adaptiveform-scribble__keyboard-sign-box
+                ELEMENT cmp-adaptiveform-scribble__geocanvas
+            ELEMENT cmp-adaptiveform-scribble__controlpanel
+                ELEMENT cmp-adaptiveform-scribble__controls
+                    ELEMENT cmp-adaptiveform-scribble__control-brush
+                        MODIFIER cmp-adaptiveform-scribble__button
+                    ELEMENT cmp-adaptiveform-scribble__brushlist
+                    ELEMENT cmp-adaptiveform-scribble__control-clear
+                        MODIFIER cmp-adaptiveform-scribble__button
+                    ELEMENT cmp-adaptiveform-scribble__control-geo
+                        MODIFIER cmp-adaptiveform-scribble__button
+                    ELEMENT cmp-adaptiveform-scribble__control-text
+                        MODIFIER cmp-adaptiveform-scribble__button
+                    ELEMENT cmp-adaptiveform-scribble__control-message
+                ELEMENT cmp-adaptiveform-scribble__controlpanel__controls
```

Comment:
we have ELEMENT cmp-adaptiveform-scribble__controlpanel
                ELEMENT cmp-adaptiveform-scribble__controls
and ELEMENT cmp-adaptiveform-scribble__controlpanel__controls as separate entities?